OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -940,7 +955,9 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
     }
 
     protected boolean isOnCourseOutline() {
-        if (courseComponentId == null) return true;
+        if (courseComponentId == null || getActivity() instanceof CourseTabsDashboardActivity) {
```

Comment:
IMO, it will always return `true` as the `CourseOutlineFragment` is using for Video xBlocks as well

Reply:
this is just to check the learner is on the Course outline screen or in the subsection screen.